sdk/communication/azure-communication-administration/azure/communication/administration/_phonenumber/_generated/models/_models.py

```diff
@@ -688,7 +688,7 @@ class PhoneNumberSearch(msrest.serialization.Model):
     """
 
     _attribute_map = {
-        'search_id': {'key': 'searchId', 'type': 'str'},
+        'reservation_id': {'key': 'searchId', 'type': 'str'},
```

Comment:
searchId to `reservation_id` as well? If it is part of the swagger, let's let the service owners know to change that as well.

Reply:
This is not a swagger change. It is a yaml change written in the [PHONE_NUMBER_SWAGGER.md](https://github.com/Azure/azure-sdk-for-python/blob/4096074feddbb76469a6816712a87fd4c6c5c1ad/sdk/communication/azure-communication-administration/swagger/PHONE_NUMBER_SWAGGER.md).

When communicating with server side, the reservation_id will be translated to search_id. When getting response from server, search_id will be changed to reservation_id. 

This is a temporary solution until the swagger change is finished by the service owners.